cloth-diapers/cloth-diapers-basics.md

```diff
@@ -0,0 +1,296 @@
+## Cloth Diapers
+
+There are many sections that need to be written up in this document, but I'll
+start by writing up the critical sections required for basic knowledge
+or those that I have a specific opinion about.
+
+_Disclaimer_: These are my personal opinions and as such, don't come at me if
```

Comment:
@iteles this disclaimer could be seen as (_unnecessarily_) undermining the validity/value of the content ... 💭  
Rather you could say: 
```suggestion
These notes are based on my research while investigating, experimenting and speaking to several mothers/parents who use cloth diapers. The notes are incomplete because this is an evolving topic. New materials, designs and workflows are being tested by us and others and there is no universally perfect setup.
```